### PR TITLE
feat: Update plugin for NativeScript 4.0

### PR DESCRIPTION
**Pull request by lini**
- bump versions of demo dependencies to target NativeScript 4
- remove plugin configuration from plugin's manifest and add it to demo
- update docs to reflect the breaking change to plugin configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +85,39 @@ inside your app project to install the module.
 </resources>
 ```
 
+##### AndroidManifest.xml
```
Comment (jogboms) [inline README.md]: I believe this is a required necessity for {N} 4.0?

Comment (lini) [reply README.md]: Yes, in 4.0 the NativeScript CLI will try to package the plugin manifest in an .aar. When the plugin manifest contains resources that are not in the plugin, the process will fail. In the case of the social login plugin, the manifest contains string resources for the Facebook configuration but the strings are supposed to be defined in the application resources. The CLI will try to package the plugin and will fail to resolve the Facebook config strings. Moving the configuration out of the plugin manifest and into the application's manifest (where the strings are) will fix this.

Comment (jogboms) [reply README.md]: Oh! Thanks. Merging then. 👍

**Comment by radeva** [comment]
Hi @jogboms, @mkloubert
When do you expect to publish a version with this fix?

**Comment by jogboms** [comment]
@radeva. I already did 2 days ago

**Pull request merged**